libs/barista-components/stacked-series-chart/src/stacked-series-chart.html

```diff
@@ -31,11 +31,17 @@
     </span>
 
     <div
-      class="dt-stacked-series-chart-track"
+      class="dt-stacked-series-chart-track dt-stacked-series-chart-{{
+        selectionMode
+      }}"
```

Comment:
Please use a binding instead of manipulating the class like this.

Reply:
This was to avoid doing 2 bindings, fix done